src/utils/openvpn.ts

```diff
@@ -377,8 +375,8 @@ export class VpnManager extends EventEmitter {
 		}
 	}
 
-	public connect(): Bluebird<true> {
-		return Bluebird.try(() =>
+	public async connect() {
+		await Bluebird.try(() =>
```

Comment:
There's no need for this `Bluebird.try` is there? You can do
```suggestion
		await this.connector
			.connect({
				port: this.mgtPort,
				shellPrompt: '',
			})
		this.emit('manager:connect');
```